utils/stack-templates/do_config.yml

```diff
@@ -0,0 +1,7 @@
+# Please read out contributing guide to learn how to properly complete this file.
+# https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md#adding-your-application
+
+# minimum_resource_requirements:
+#   node_count: 2
+#   cpu: "500m"
```

Comment:
We should uncomment this and populate it with the minimum resource cluster from DigitalOcean.
In that case the user would have to just update the values.